sdk/azidentity/azidentity.go

```diff
@@ -25,6 +27,31 @@ const (
 	defaultSuffix = "/.default"
 )
 
+// log environment variables that can be used for credential types
+func logEnvVars() {
+	// Log available environment variables
```

Comment:
You might want to add the following to the beginning of this func.
```go
if !Log().Should(LogCredential) {
    return
}
```
This will skip the following code if logging is disabled.

Reply:
Done